src/App.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component } from "react";
+
+import TodoItemsList from "./components/TodoItemsList";
+import TodoInput from "./components/TodoInput";
+import TodoSearch from "./components/TodoSearch";
+import "./App.css";
+const uuidv4 = require('uuid/v4');
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      surname: "",
+      message: "",
+      isSubmitted: false,
```

Comment:
please make sure to remove items that are not used anywhere e.g. `isSubmitted`, `key`